### DISCUSSION TITLE
Zero Export script embedded in Open DTU

### DISCUSSION THREAD

**Discussion by rradkov** [tbnobody/OpenDTU#2222] [Ideas]
Hello everyone. 
Quick question. I saw that OpenDTU-Onbattery support smart meter and also support Zero Export script but there is limitation only to one Inverter. Is there a possibility that you implement the same features here as well - Zero Export script running on DTU and smart meter ? It will be really beneficial to all users and one device less to use in order to have good and cheap open source photovoltaic system

**Comment by Tpaydt** [comment]
Maybe this will interest you 

https://github.com/helgeerbe/OpenDTU-OnBattery/pull/1216
